File: Selection statement/Question4/question4.cpp
#include <iostream>

int main()
{
    int num;

    std::cout << "Enter a number: " << std::endl;
    std::cin >> num;

    if (num > 0)
    {
        std::cout << num << " is a positive number." << std::endl;
    }
    else if (num < 0)
    {
        std::cout << num << " is a negative number." << std::endl;
    }
    else
    {
        std::cout << num << " is zero." << std::endl;
    }
}

File: Selection statement/Question4/question4.java
import java.util.Scanner;

public class question4 {
    public static void main(String[] args) {
        try (Scanner scan = new Scanner(System.in)) {
            int num;

            System.out.println("Enter a number: ");
            num = scan.nextInt();

            if (num > 0) {
                System.out.print(num + " is positive");
            } else if (num < 0) {
                System.out.print(num + " is negative");
            } else {
                System.out.print(num + " is zero");
            }
        }
    }
}
